packages/swapper/src/swappers/ChainflipSwapper/endpoints.ts

```diff
@@ -279,21 +279,20 @@ export const chainflipApi: SwapperApi = {
   checkTradeStatus: async ({
     config,
     quoteId,
+    swap,
   }): Promise<{
     status: TxStatus
     buyTxHash: string | undefined
     message: string | [string, InterpolationOptions] | undefined
   }> => {
-    const swap = tradeQuoteMetadata.get(quoteId)
-    if (!swap) throw Error(`Missing trade quote metadata for quoteId ${quoteId}`)
-    // Note, the swapId isn't the quoteId - we set the swapId at pre-execution time, when getting the receive addy and instantiating a flip swap
-    const swapId = swap.id
+    const chainflipSwapId = tradeQuoteMetadata.get(quoteId)?.id || swap?.metadata.chainflipSwapId
+    if (!chainflipSwapId) throw Error(`Missing trade quote metadata for quoteId ${quoteId}`)
```

Comment:
i.e stop referencing `quoteId` 
```suggestion
    if (!chainflipSwapId) throw Error(`swapId is required`)
```

Reply:
Done in [fbcd366](https://github.com/shapeshift/web/pull/9625/commits/fbcd366ded019ee7c8586022801e2a9625cc80dd)